scylla-cql/src/frame/response/result.rs

```diff
@@ -558,6 +558,16 @@ impl ResultMetadata {
             col_specs,
         }
     }
+
+    #[inline]
+    pub fn col_count(&self) -> usize {
+        self.col_count
+    }
+
+    #[inline]
+    pub fn col_specs(&self) -> &[ColumnSpec] {
+        &self.col_specs
+    }
```

Comment:
Could you use `ColumnSpec<'_>` to signify that lifetime is not static?

Reply:
Actually, what lifetime does the current syntax imply? How does it differ from `'_` ?

Reply:
Current syntax does imply `'_` lifetime - I did not ask for a change in semantics, but for highlighting current semantics.